src/main/java/sophisticated_wolves/core/SWConfiguration.java

```diff
@@ -35,7 +39,8 @@ public class SWConfiguration {
 
         CUSTOM_WOLF_TEXTURES = BUILDER.comment("Should sophisticated wolves use custom textures(or be like vanilla wolves)?")
                 .define("Use Custom Wolf Textures", true);
-        //TODO remove ???
+        //TODO i still think some people kinda want this ... should remain as an option that people could always
+        //disable. Or.... DoggyTalents could be an addon for this, with dog bed usage. 
 //        RESPAWNING_WOLVES = BUILDER.comment("Should sophisticated wolves respawn(and despawn for non tamed)?")
 //                .define("Respawning Wolves", true);
```

Comment:
Despawn of wild wolves were removed in minecraft 1.10, so the main purpose of this option has no sence anymore. This is why I'm going to remove it.

Reply:
No no, i mean the Respawning part ...

Reply:
Btw if you want you can go ahead and remove the comment.

Reply:
This option was added long time ago to be able to disable/enable wolves despawn and respawn. As far as they do not despawn any more there are no reasons to respawn them. Furthermore it is possible to buy new wolves in village now, as an alternative way to obtain wolf.

Reply:
Okay i get it, i thought you mean the reviving mechanic like Doggy talents 's /dog revive ......